samples/AspireWithPython/AspireWithPython.AppHost/Program.cs

```diff
@@ -0,0 +1,6 @@
+﻿var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddPythonProject("instrumented-python-project", "..\\InstrumentedPythonProject", "main.py")
```

Comment:
NIT: .NET supports both backslash and forward slash, I prefer to standardize on forward slash as it's less characters to type and it doesn't require an escape sequence.

```suggestion
builder.AddPythonProject("instrumented-python-project", "../InstrumentedPythonProject", "main.py")
```